src/test/regress/failure_schedule

```diff
@@ -5,17 +5,17 @@ test: failure_test_helpers
 test: failure_setup
 test: multi_test_helpers
 
-test: failure_ddl
-test: failure_truncate
-test: failure_create_index_concurrently
-test: failure_add_disable_node
-test: failure_copy_to_reference
-test: failure_copy_on_hash
-test: failure_create_reference_table
-test: failure_create_distributed_table_non_empty
-test: failure_create_table
-
-test: failure_1pc_copy_hash
-test: failure_1pc_copy_append
-test: failure_multi_shard_update_delete
+# test: failure_ddl
+# test: failure_truncate
+# test: failure_create_index_concurrently
+# test: failure_add_disable_node
+# test: failure_copy_to_reference
+# test: failure_copy_on_hash
+# test: failure_create_reference_table
+# test: failure_create_distributed_table_non_empty
+# test: failure_create_table
+#
+# test: failure_1pc_copy_hash
+# test: failure_1pc_copy_append
+test: failure_single_select
```

Comment:
This hunk confuses me a lot,
 - we should not comment out all existing tests
 - not remove existing tests
 - I would expect this to be reduced to only the addition of `failure_ref_tables`.

Reply:
I guess by now you've noticed my hygiene around committing or reverting changes to this file is very poor :-P

I comment out other tests when I'm working because I want to avoid the extra time running them, then often forget to revert or commit my new test. Sorry!